### DISCUSSION TITLE
"Your account has been flagged."

### DISCUSSION THREAD

**Discussion by karnull** [community/community#32007] [Other Feature Feedback, Questions, & Ideas]
I take a break from coding for a few days, and come back only to see that my account has been flagged. 

Why? 

I haven't done anything to deserve this. I don't even use github for anything but to `git push`, yet this happened. Sure, it is possible for the bot to detect some false alarm, even though that's highly unlikely, but even then its been over 10 days since I've composed an email about this to the "contact github" but haven't received no response whatsoever.

From what I could google about "account getting flagged", this can be fixed by changing primary email to some @outlook.com email BUT IS THAT REALLY THE CASE? Is this a punishment for not using microsoft email?

**Comment by awebmekhilef** [comment]
lmao
